clickhouse_search/fixtures/clickhouse_search.json

```diff
@@ -351,7 +407,6 @@
       "filters": [],
       "sign": 1,
       "calls": [
-        ["NA20872", 1, 99, 0.34285714285714286, 35],
```

Comment:
this call has no corresponding sample in the postgres database and was included in the fixture data to check that it was properly ignored, but now needs removal as this behavior has changed